lib/compliance/triggers/triggers.js

```diff
@@ -4,6 +4,8 @@ const BN = require('../../bn')
 const { TRIGGER_TYPES, REQUIREMENTS, DIRECTIONS } = require('./consts')
 
 const filterSinceThreshold = days => _.filter(it => daysSince(it.created) < days)
+// BACKWARDS_COMPATIBILITY 7.5
+// to be deleted: no need to filter as we're doing it on the server-side
```

Comment:
I think this is enough, as the filter result theoretically will be equal with the starting list of txs, but we can go up a notch and ignore the filtering completely if we want, to save computer resources. Confirm with Rafael if this is something we want

```suggestion
// BACKWARDS_COMPATIBILITY <relevant_version_here>
// to be deleted: no need to filter as we're doing it on the server-side
let filterNoFiatTx
const serverVersion = this.trader.serverVersion
if (!serverVersion || semver.lt(serverVersion, <relevant_version_here>)) {
  filterNoFiatTx = _.filter(it => BN(0).lt(it.fiat || 0))
} else {
  filterNoFiatTx = originalTxs // This line can be done in multiple different ways, find the most elegant one
}
```

Reply:
No need for this, complicates the code for low upside.